### PR TITLE
Documentation - tacho motors and XV11 references section

### PR DESCRIPTION
**Pull request by bmegli**

### CODE REVIEW

File: docs/tutorials/tacho-motors.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ It should be somewhere close to the value returned by the `count_per_rot`
 
     $ cat $MC/count_per_rot
     360
-
+
```
Comment (dlech) [inline docs/tutorials/tacho-motors.md]: why add tab here?

File: docs/tutorials/tacho-motors.md [hunk 2]
```diff
@@ -92,6 +92,8 @@ it very fast). Press `^C` when you are done. Note: `\033[0G` is an escape code
 that moves the cursor back to the beginning of the line so that we don't fill up
 the screen with numbers.
 
+The range of `position` attribute is between -2,147,483,648 and +2,147,483,647. It is unlikely that you will ever reach the limit.
```
Comment (dlech) [inline docs/tutorials/tacho-motors.md]: Might be good to add the unit of measurement here. It is "tachometer counts" (not degrees).

File: docs/tutorials/tacho-motors.md [hunk 3]
```diff
@@ -151,6 +153,8 @@ change.
     $ echo run-to-abs-pos > $MC/command
     $ while true; do echo -en "\033[0G$(cat $MC/position)   "; done
 
+Note: For `run-to-abs-pos` command only the absolute value of `duty_cycle_sp` or `speed_sp` matters. The direction of movement is determined automatically.
```
Comment (dlech) [inline docs/tutorials/tacho-motors.md]: :+1: we should add this to the source code documentation as well. 

* https://github.com/ev3dev/lego-linux-drivers/blob/master/motors/tacho_motor_class.c#L75
* https://github.com/ev3dev/lego-linux-drivers/blob/master/motors/tacho_motor_class.c#L123

**Comment by dlech** [comment]
Thanks! Also a tip. GitHub doesn't notify when you push more commits to a pull request. Be sure to make a comment when you do this or I may forget to check.

**Comment by bmegli** [comment]
Ok!

**Pull request merged**